tv_apps/fling-player/js/connector.js

```
@@ -75,6 +77,24 @@
     mDBG.log('Connector#sendMsg');
     mDBG.log('msg = ', msg);
     this._connection.send(castingMessage.stringify(msg));
+    var txt = castingMessage.stringify(msg);
```

How about doing try/catch here rather than just suppress the error?